mne/io/ant/tests/test_ant.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import annotations
+
+import warnings
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+import pytest
+from numpy.testing import assert_allclose
+
+from mne import Annotations
+from mne.io import BaseRaw, read_raw_ant, read_raw_brainvision
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+
+def read_raw_bv(fname: Path) -> BaseRaw:
+    """Read a brainvision file exported from eego."""
+    with warnings.catch_warnings():
+        warnings.filterwarnings(
+            "ignore",
+            message="Limited .* annotation.*outside the data range.",
+            category=RuntimeWarning,
+        )
+        raw_bv = read_raw_brainvision(fname)
+    return raw_bv
+
+
+@pytest.fixture(scope="module")
+def ca_208() -> dict[str, dict[str, Path]]:
+    """Return the paths to the CA_208 dataset containing 64 channel gel recordings."""
+    pytest.importorskip("antio", minversion="0.2.0")
+
+    from antio.datasets import ca_208
+
+    data_path = ca_208.data_path()
+    cnt = {
+        "short": data_path / "test_CA_208.cnt",
+        "amp-dc": data_path / "test_CA_208_amp_disconnection.cnt",
+        "start-stop": data_path / "test_CA_208_start_stop.cnt",
+    }
+    bv = {key: value.with_suffix(".vhdr") for key, value in cnt.items()}
+    return {"cnt": cnt, "bv": bv}
```

Comment:
Might need to add a pytest skip if the files did not get correctly downloaded by `pooch`.

Reply:
I don't think we ever really want to download files in tests. Can we add them to `mne-testing-data` instead?

Reply:
It's only a couple of small files which are also used in tests in `antio`. It would be a bit excessive to download the entire MNE testing dataset just for that in `antio`, that's why I wanted to provide them separately with `pooch`.

Reply:
I'd rather then also duplicate them in MNE-Python. Here once you download `mne-testing-data` you should have all the files needed for tests. I think the tiny bit of file duplication is worth the simplicity of sticking with the current downloading infrastructure here

Reply:
Thinking out loud, that means running `pytest mne/io/ant/tests` in a workflow in `antio` will require again the entire `testing` dataset from MNE.
I'll think about it a bit more to see if there is an elegant way to avoid this download, else I'll just add it with a caching step when I move the files to MNE's `testing` dataset.

Reply:
Yes but with caching it's not too bad actually. 

You could also trick MNE into thinking that the whole testing dataset is there. A properly named folder (`~/mne_data/MNE-testing-data`) containing a version.txt with a version number inside it will do it. Then you just make sure that in addition to those the actual files you need are there you're good.